x/ethbridge/client/cli/tx.go

```diff
@@ -106,6 +120,17 @@ func GetCmdLock(cdc *codec.Codec) *cobra.Command {
 
 			txBldr := authtxb.NewTxBuilderFromCLI().WithTxEncoder(utils.GetTxEncoder(cdc))
 
+			ethereumChainID, err := strconv.Atoi(viper.GetString("ethereum-chain-id"))
+			if err != nil {
+				return err
+			}
+
+			tokenContractString := viper.GetString("token-contract-address")
```

Comment:
ditto on using the constant

Reply:
https://github.com/cosmos/peggy/commit/7e1201a2ec806b75a086bced38f059101798adb5